src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -2957,25 +2957,26 @@ def wait_for_queue_to_drain(topic_name, tenant=None, account=None, http_port=Non
     log.info('waited for %ds for queue %s to drain', time_diff, topic_name)
 
 
-@attr('basic_test')
+@attr('kafka_test')
```

Comment:
can you please also add an http version of this?
we would need to fix http (eventually), so we should have a test to verify our fix